exporter/opentelemetry-exporter-prometheus-remote-write/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Changelog
+
+## Unreleased
```

Comment:
I would include "initial release" and the PRs associated.

Reply:
Added Initial Release in CHANGELOG and will update it with each subsequent PR.